config/brakeman.yml

```diff
@@ -0,0 +1,3 @@
+---
+:url_safe_methods:
+- :show_link_args
```

Comment:
Tells brakeman that `show_link_args` generates a safe url. See https://brakemanscanner.org/docs/warning_types/link_to_href/

This is one way to deal with the Codeclimate/brakeman issue [Potentially unsafe model attribute in link_to href](https://github.com/MushroomObserver/mushroom-observer/pull/718#issuecomment-743769794). But **it is a valid fix only if the brakeman issue is a false positive**.

Two other ideas for dealing with this issue:
1. Change ObjectLinkHelper#location_link line 31 as follows. (This does not assume that the issue was a false positive.)
From:
```ruby
      link_to(link_string, location.show_link_args)
```
To:
```ruby
      link_to(link_string, "#{location_show_location_path}/#{location.id}")
```

2. Just tell Codeclimate "Invalid" or "Wontfix"